tests/test_crawlers.py

```diff
@@ -599,3 +602,69 @@ def test_iterating(self):
         with self.assertLogs(self.crawler.LOGGER):
             for i, url in enumerate(self.crawler):
                 self.assertEqual(url, expected_urls[i])
+
+
+class FTPCrawlerTestCase(unittest.TestCase):
+    """Tests for the FTP crawler"""
+
+    def the_503_fake_ftp_exception(self, *args):
```

Comment:
Use verb in function name. Please correct below as well.
```suggestion
    def raise_exception_503(self, *args):
```

Reply:
Done in the next commit